public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Cancun-Deneb (Dencun) FAQ
+description: Frequently asked questions regarding the Cancun-Deneb (Dencun) network upgrade
+lang: en
+---
+
+# Cancun-Deneb (Dencun) {#dencun}
+
+Cancun-Deneb (Dencun) is an upgrade to the Ethereum network which activates **Proto-Danksharding (EIP-4844)**, introducing data **blobs** for cheaper layer 2 (L2) rollup storage.
+
+A new transaction type enables rollup providers to store data in a more cost-effective manner, in what are known as "blobs", which are guaranteed to be made available to the network, but can be discarded after a couple weeks.
+
+This significantly reduces the cost of using rollups, limits chain growth, and helps to support more users while maintaining security and a decentralized set of node operators.
+
+## When do we expect that rollups will reflect lower fees as a result of Proto-Danksharding? {#when}
+
+- This upgrade is set for activation at the start of [epoch](/glossary/#epoch) 269568, occurring on **13-Mar-2024 (UTC)**
+- All major rollup providers, such as Arbitrum or Optimism, have signaled that blobs will be supported immediately following the upgrade
+- Individual rollup support may vary as each rollup must upgrade to take advantage of the new blob space
+
+## How can ETH be converted after the hard fork? {#scam-alert}
+
+- **There is NO need to convert your ETH**. Your account balances will not change, and the ETH you hold will continue to be available in the same form after the upgrade (hard fork).
+- <Emoji text="⚠️" /> <strong>Anyone telling you to upgrade your ETH is attempting to scam you!</strong> Users do not need to do anything for this upgrade, and your assets will remain unaffected.
+
+[More on recognizing and avoiding scams](/security/)
+
+## What problem is the Dencun network upgrade solving? {#network-impact}
+
+Dencun primarily addresses **scalability** (handling more users and more transactions) with **affordable fees**, while **maintaining decentralization** of the network.
+
+The Ethereum community has been taking a "rollup-centric" approach to its growth, which places layer 2 rollups as the primary means to safely support more users.
+
+Rollup networks handle the _processing_ (or "execution") of transactions separate from Mainnet, and then publish a cryptographic proof of the results back to Mainnet for record keeping. Storing these proofs carries an expense (in the form of [gas](/glossary/#gas)), which prior to Proto-Danksharding had to be stored permanently by all network node operators, making it an expensive task.
```

Comment:
This may be too in the weeds, but ZKRs publish a proof while ORs published compressed transaction data.